### PR TITLE
Added functionalities to create policy and role

### LINKED ISSUE #8: Spin the instance with an instance profile
For additional information:
https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html

### PR DESCRIPTION
**Pull request by yerdreic**
- Added 2 files under the assets dir, which are the policy and iam role json files
- Added the functionality or the creation of the policy and role
- Added the role to instance profile

### CODE REVIEW

File: ec2_instances/assets/policy.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+            "Action": [
+                "directconnect:Describe*",
+                "directconnect:List*",
+                "ec2:DescribeVpnGateways",
+                "ec2:DescribeTransitGateways"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
```
Comment (lmilbaum) [inline ec2_instances/assets/policy.json]: Please add end of line

File: ec2_instances/config.sh [hunk 2]
```diff
@@ -7,9 +7,12 @@ SG_NAME="poc-aws"
 SG_TAG_KEY="Name"
 SG_TAG_VAL="poc-aws"
 SG_ID=""
-UBUNTU_X86_ID="ami-0caef02b518350c8b"
+IMAGE_ID="ami-0caef02b518350c8b"
```
Comment (lmilbaum) [inline ec2_instances/config.sh]: Is this change related to the PR scope?

File: ec2_instances/create_ec2.sh [hunk 3]
```diff
@@ -12,7 +33,7 @@ chmod 600 $SSH_PRIVATE_FILE_NAME
 
 IS_SG_EXISTS=$(aws ec2 describe-security-groups --filters "Name=tag:$SG_TAG_KEY,Values=$SG_TAG_VAL" --query "SecurityGroups[*].{Name:GroupName}" --output text)
 
-if [[ "$IS_SG_EXISTS" == $SG_NAME ]]; then
+if [[ "$IS_SG_EXIST" == $SG_NAME ]]; then
```
Comment (lmilbaum) [inline ec2_instances/create_ec2.sh]: Is this change related to PR scope?

File: ec2_instances/create_ec2.sh [hunk 4]
```diff
@@ -22,7 +43,7 @@ else
 fi
 
 aws ec2 run-instances \
---image-id $UBUNTU_X86_ID \
+--image-id $IMAGE_ID \
```
Comment (lmilbaum) [inline ec2_instances/create_ec2.sh]: same question as above

**Comment by lmilbaum** [comment]
Was the script validated by running it?

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: ec2_instances/config.sh (hunk 2 above)
Comment (yerdreic) [reply ec2_instances/config.sh]: Nope. I used this PR to fix some other syntaxes changes. Probably not the best practice. Will separate it

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**